getting-started/spring-rest-guide/README.MD

```diff
@@ -1 +1 @@
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/spring-rest-guide/devfile.yaml)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/spring-rest-guide/devfile.yaml)
```

Comment:
verified workspace startup, commands and LS basic functionality - OK
The app in the devfile although doesn't seem healthy.
![Screenshot from 2021-02-22 12-52-45](https://user-images.githubusercontent.com/29278800/108705060-1a3a8700-750d-11eb-9765-b8225ff36685.png)